src/pages/lists/status-list/index.tsx

```diff
@@ -1,78 +1,371 @@
+    },
+    time: {
+        fontWeight: 700,
+        fontSize: '12px',
+    },
+    timePeriod: {
+        marginLeft: '4px',
+    },
+    rightComponentRoot: {
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+    },
+    assignedTag: {
+        marginLeft: theme.spacing(4),
+        marginTop: 4,
+    },
+    activeTag: {
+        marginLeft: 0,
+        marginTop: 4,
+    },
+    listItemTag: {
+        marginLeft: theme.spacing(2),
+        marginRight: theme.spacing(4),
+    },
+    rightComponentChevron: {
+        color: colors.gray[500],
+    },
+    infoComponent: {
+        display: 'flex',
+        margin: `4px 0px 4px ${theme.spacing(4)}px`,
     },
 }));
 
-const createInfoListItemConfig = (index: number, randomStatus: string, tag?: boolean): InfoListItemProps => {
+const getTitle = (
+    deviceStatus: string,
+    device: string,
+    hasTimeStamp: boolean,
+    isMobile = false,
+    classes: Record<string, any>
+): ReactNode => (
+    <div className={clsx(classes.listItemTitle, !hasTimeStamp ? classes.listItemTitleWithoutTimeStamp : '')}>
+        <Typography variant={'subtitle1'} noWrap>
+            {deviceStatus}
+        </Typography>
+        {!isMobile && (
+            <Typography variant={'body1'} noWrap>
+                : &nbsp;{device}
+            </Typography>
+        )}
+    </div>
+);
+
+const getSubtitle = (
+    station: string,
+    location: string,
+    hasTimeStamp: boolean,
+    isMobile = false,
+    classes: Record<string, any>
+): string | Array<string | JSX.Element> | undefined => [
+    <span key="station" className={clsx(classes.station, !hasTimeStamp ? classes.stationWithoutTimeStamp : '')}>
+        {station}
+    </span>,
+    !isMobile ? (
+        <span key="location" className={classes.location}>
+            {`<`} &nbsp; {location}
+        </span>
+    ) : (
+        ''
+    ),
+];
+
+const getLeftComponent = (
+    time: string,
+    timePeriod: 'AM' | 'PM',
+    date: string,
+    hasIcon: boolean,
+    classes: Record<string, any>
+): ReactNode => (
+    <div className={clsx(classes.leftComponentRoot, !hasIcon ? classes.leftComponentRootWithoutIcon : '')}>
+        <div className={classes.timeStamp}>
+            <Typography classes={{ root: classes.time }}>{time}</Typography>
+            <Typography variant={'caption'} classes={{ root: classes.timePeriod }}>
+                {timePeriod}
+            </Typography>
+        </div>
+        <Typography variant={'caption'}>{date}</Typography>
+    </div>
+);
+
+const getRightComponent = (isMobile: boolean, tag: boolean, classes: Record<string, any>): ReactNode => (
+    <div className={classes.rightComponentRoot}>
+        {tag && !isMobile && (
+            <>
+                <ListItemTag label={'assigned'} backgroundColor={colors.blue[500]} />
+                <ListItemTag
+                    label={'active'}
+                    backgroundColor={colors.red[500]}
```

Comment:
again, avoid using @pxblue/colors directly here. Usually you can pull things out of theme.palette... except that orange color on this page.

You can take a peek at MUI's default theme here: https://mui.com/customization/default-theme/